bin/pit-init.sh

```diff
@@ -79,18 +84,43 @@ echo 'Generating Configuration ...'
     popd
 )
 
-SHASTA_CFG_DIR=$PREP_DIR/site-init
-if [ ! -d $SHASTA_CFG_DIR ] ; then
-    echo >&2 "Need $SHASTA_CFG_DIR; this needs to be created by hand if this is a first-time install."
+SITE_INIT=$PREP_DIR/site-init
+site_init_error=0
+if [ ! -d $SITE_INIT ] ; then
```

Comment:
This should also come before running `csi config init`.   We should check for all required data and tools before we start making changes.